packages/svelte/src/internal/client/reactivity/sources.js

```diff
@@ -191,17 +191,12 @@ export function internal_set(source, value) {
 			is_runes() &&
```

Comment:
is the comment above this if block still correct? From looking at the code it doesn't look like it's only running when it's the first time